File: server/src/main/java/by/brawl/ws/holder/BodypartHolder.java
package by.brawl.ws.holder;

import by.brawl.entity.bodypart.Bodypart;
import by.brawl.entity.bodypart.BodypartType;

public class BodypartHolder {
    private String id;
    private BodypartType type;

    public BodypartHolder(Bodypart bodypart) {
        id = bodypart.getId();
        type = bodypart.getType();

    }

    public String getId() {
        return id;
    }

    public BodypartType getType() {
        return type;
    }
}

File: server/src/main/java/by/brawl/ws/holder/BodypartHolder.kt
package by.brawl.ws.holder

import by.brawl.entity.bodypart.Bodypart
import by.brawl.entity.bodypart.BodypartType

class BodypartHolder(bodypart: Bodypart) {
    val id = bodypart.id
    val type = bodypart.type
}
